src/store/actions.js

```diff
@@ -98,6 +104,10 @@ export default {
     let scores = context.getters.getPlayerScores()
     for (let scoreInfo of scores) {
       if (scoreInfo.score >= context.state.scoreLimit) {
+        //Winmer Info added on log
```

Comment:
Misspelt "winner"

Reply:
i resolved all the issues, but i created test branch from master branch and it doesn't have scan card. i can't check this issue 
"Playing "Scan" shows as Overclock in the log"

Reply:
We can gov over this tomorrow then.